tensorflow/api_template_v1.__init__.py

```diff
@@ -192,6 +187,18 @@ def _running_from_pip_package():
       # Load Pluggable Device Library
       _ll.load_pluggable_device_library(_plugin_dir)
 
+# Explicitly import lazy-loaded modules to support autocompletion.
+# pylint: disable=g-import-not-at-top
+if _typing.TYPE_CHECKING:
+  from tensorflow_estimator.python.estimator.api._v1 import estimator as estimator
+  from keras.api._v1 import keras
+  from keras.api._v1.keras import losses
+  from keras.api._v1.keras import metrics
+  from keras.api._v1.keras import optimizers
+  from keras.api._v1.keras import initializers
+  from tensorflow import contrib
```

Comment:
not 100% sure this is the correct import for `contrib`
`from tensorflow import contrib`

Reply:
Let's try without it. We'll definitely need to manually alter this PR inside Google

Reply:
removed contrib import